ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { PageSection } from '@patternfly/react-core';
+
+import PageNotFound from 'Components/PageNotFound';
+import PageTitle from 'Components/PageTitle';
+
+import {
+    vulnerabilitiesWorkloadCveDeploymentSinglePath,
+    vulnerabilitiesWorkloadCveImageSinglePath,
+    vulnerabilitiesWorkloadCveSinglePath,
+    vulnerabilitiesWorkloadCvesPath,
+} from 'routePaths';
+import WorkloadCvesDeploymentSinglePage from './WorkloadCvesDeploymentSinglePage';
+import WorkloadCvesImageSinglePage from './WorkloadCvesImageSinglePage';
+import WorkloadCvesOverviewPage from './WorkloadCvesOverviewPage';
+import WorkloadCvesSinglePage from './WorkloadCvesSinglePage';
+
+function WorkloadCvesPage() {
+    return (
+        <Switch>
+            <Route path={vulnerabilitiesWorkloadCveSinglePath} component={WorkloadCvesSinglePage} />
+            <Route
+                path={vulnerabilitiesWorkloadCveImageSinglePath}
+                component={WorkloadCvesImageSinglePage}
+            />
+            <Route
+                path={vulnerabilitiesWorkloadCveDeploymentSinglePath}
+                component={WorkloadCvesDeploymentSinglePage}
+            />
+            <Route
+                exact
+                path={vulnerabilitiesWorkloadCvesPath}
+                component={WorkloadCvesOverviewPage}
+            />
+            <Route>
```

Comment:
We'll likely want to use another component for 404s down the line as a follow up.

e.g. For situations where a user has a deployment single page bookmarked, but the deployment no longer exists in the system.

Reply:
Yes, if user has permission and path (not including id) is valid, then more specific wording and link is better.